src/Hooks/customHooks.js

```diff
@@ -3,9 +3,8 @@ import { useLocation } from 'react-router-dom';
 // Custom hook for query string parsing from URL
 // Useful for things like Search, which normally can't depend
 // only on a text input (i.e. someone might want a shareable URL)
-export const useQuery = (urlPath, param) => {
-  // `urlPath` is the path string that comes after "{BASE_URL}/"
-  // `param` is the key of the query parameter whose value we want
-  const path = useLocation()[urlPath];
+// `param` is the key of the query parameter whose value we want
+export const useQuery = param => {

```

Comment:
To anyone who's wondering - `useLocation()` always has a field called `search`. I thought that this was specific to the route you were on, so I thought you needed to pass in a parameter for that (it just so happened my previous work was on `/search`). 

Anyway, I was very wrong! Please forgive the stupidity haha

Reply:
shhhh its fine you're learning

Reply:
thanks for the explanation 👍